### PR TITLE
feat: changed reviews text and on click behaviour

### PR DESCRIPTION
**Pull request by rbadici-plenty**

### CODE REVIEW

**Review by FabianGerke — CHANGES_REQUESTED** [review]
Fix the lint errors first
Command: yarn lint

File: apps/web/components/ui/PurchaseCard/PurchaseCard.vue [hunk 1]
```diff
@@ -20,8 +20,8 @@
     <div class="inline-flex items-center mt-4 mb-2">
       <SfRating size="xs" :value="productGetters.getAverageRating(reviewAverage)" :max="5" />
       <SfCounter class="ml-1" size="xs">{{ productGetters.getTotalReviews(reviewAverage) }}</SfCounter>
-      <SfLink href="#" variant="secondary" class="ml-2 text-xs text-neutral-500">
-        {{ $t('reviewsCount', { count: productGetters.getTotalReviews(reviewAverage) }) }}
+      <SfLink href="#customerReviewsAccordion" variant="secondary" class="ml-2 text-xs text-neutral-500">
+        {{ $t('showAllReviews') }}
```
Comment (FabianGerke) [inline apps/web/components/ui/PurchaseCard/PurchaseCard.vue:24]: Why not using the smooth solution from the pwa?

`const scrollToReviews = () => {
      document
        .getElementById('customerReviewsAccordion')
        .scrollIntoView({ behavior: 'smooth', block: 'start', inline: 'nearest' });
    };`

**Review by FabianGerke — CHANGES_REQUESTED** [review]
Maybe also an idea to open the review accordion when clicking on "Show all reviews"?
@maxiroellplenty

File: apps/web/components/ui/PurchaseCard/PurchaseCard.vue [hunk 2]
```diff
@@ -80,13 +68,36 @@ const { product } = toRefs(props);
...
+  const headerElementOffset = headerElement.offsetHeight ?? 0;
+
+  window.scrollTo({
+    top: customerReviewsAccordionElementOffset - headerElementOffset,
+    behavior: 'smooth',
+  });
+};
+
+const scrollToReviews = () => {
+  openReviewsAccordion();
```
Comment (ksted) [inline apps/web/components/ui/PurchaseCard/PurchaseCard.vue:100]: **todo**: Check if already open

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**